### PR TITLE
Simplify the isolated host Docker setup instruction

### PR DESCRIPTION
**Pull request by mmnosek**
Since we have more methods to start the container using docker, exacts steps should be not described in "isolated host" section but rather linked to the list of available methods.

### CODE REVIEW

**Review by rasika-chivate — APPROVED** [review]

File: docs/setting-up/server/docker.md [hunk 1]
```diff
@@ -429,24 +429,7 @@ If the host where you will run PMM Server has no internet connection, you can do
...
-    ```sh
-    docker run \
-    --detach \
-    --restart always \
-    --publish 443:443 \
-    --volumes-from pmm-data \
-    --name pmm-server \
-    percona/pmm-server:{{release}}
-    ```
+6. [Run the container](#run) as if your image is already pulled.
```
Comment (shoffman-percona) [inline docs/setting-up/server/docker.md:432]: 6. [Run the container](#run) as if your image is already pulled using your desired method for a storage volume (you can step over any `docker pull` commands as the image has been pre-staged)

**Review by shoffman-percona — CHANGES_REQUESTED** [review]
Tried to make it a little more clear that you can choose your method and just ignore the docker pull commands.

File: docs/setting-up/server/docker.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/setting-up/server/docker.md]: Updated.

**Comment by rasika-chivate** [comment]
> Tried to make it a little more clear that you can choose your method and just ignore the docker pull commands.

Updated

**Review by shoffman-percona — APPROVED** [review]
Looks better now!

**Pull request merged**